Argo/Types/Decoded.swift

```diff
@@ -31,6 +31,16 @@ public extension Decoded {
   }
 }
 
+extension Decoded: Printable {
+  public var description: String {
+    switch self {
+    case let .Success(x): return x.description
+    case let .TypeMismatch(s): return s
+    case let .MissingKey(s): return s
```

Comment:
Do we want these values wrapped in their enum case? That would mimic the behavior of `Optional`, `Result`, etc.